lib/chef/knife/vsphere_datastore_list.rb

```diff
@@ -18,23 +18,6 @@
 require 'chef/knife'
 require 'chef/knife/base_vsphere_command'
 
-def number_to_human_size(number)
```

Comment:
I also saw that the base class was calling it and thought that it may have been pulled in from one of the knife classes, I can add it to the base class if that makes sense, so that it does not need to be defined over and over